controllers/controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchCategories } = require("../models/model");
+
+exports.getCategories = (req, res) => {
+  fetchCategories().then((categoryArray) => {
+    res.send(categoryArray);
+  });
+};
+
+exports.catchAll = (req, res) => {
+  res.status(404).send({ msg: "link not found" });
+};
```

Comment:
move this error helping middleware to a different file, it is not being used as a controller in the sense that the other function is.